CMakeLists.txt

```diff
@@ -294,29 +295,47 @@ set(CMAKE_SKIP_BUILD_RPATH  FALSE)
 set(CMAKE_BUILD_WITH_INSTALL_RPATH FALSE)
 
 set(CMAKE_MACOSX_RPATH 1)
-set(CMAKE_BUILD_RPATH "${AUTOBUILD_INSTALL_PATH}/bin;${AUTOBUILD_INSTALL_PATH}/lib;${AUTOBUILD_INSTALL_PATH}/lib64")
+set(CMAKE_BUILD_RPATH "./;${AUTOBUILD_INSTALL_PATH}/bin;${AUTOBUILD_INSTALL_PATH}/lib;${AUTOBUILD_INSTALL_PATH}/lib64")
```

Comment:
Why was this change necessary? I'm going to try to undo this change, and experiment with a few other changes on builds on the Mac Travis.

Reply:
It was partly to address the issue with the shared library not finding libraries located in the same directory, which was an issue with the shared_library for some of the interfaces.  basically include the current path in the RPATH.